### ISSUE TITLE
Implement responsive hamburger menu

### ISSUE THREAD

**Issue by ellenmacpherson** [b30wulffz/moviefy#2]
Implement an accessible, responsive hamburger menu for mobile and tablet devices. This should be made accessible for screen readers through the use of aria labels.

**Comment by ellenmacpherson** [comment]
I thought it would be useful, since Hacktoberfest is on its way, to break down the responsive issue into smaller tasks. This way no-one is working on a duplicate issues and there should be fewwer merge conflicts.
